include/Dollar.h

```diff
@@ -5,9 +5,9 @@
 class Dollar                                                                                              
 {
 public:
-int amount;                                                                                   
+    int amount;                                                                                   
```

Comment:
You forget the methods. They should be indented as well. After fixing this you're all set for approval :+1: